src/ci/github-actions/ci.yml

```diff
@@ -73,15 +73,23 @@ x--expand-yaml-anchors--remove:
   - &base-job
     env: {}
 
-  - &job-linux-xl
+  - &job-linux-8c
+    os: ubuntu-20.04-8core-32gb
+    <<: *base-job
+
+  - &job-linux-16c
     os: ubuntu-20.04-16core-64gb
     <<: *base-job
 
   - &job-macos-xl
     os: macos-12-xl
     <<: *base-job
 
-  - &job-windows-xl
+  - &job-windows-8c
+    os: windows-2019-8core-32gb
+    <<: *base-job
+
+  - &job-windows-16c
```

Comment:
This seems incorrect - it's named 16c but has 8 cores?
Since it isn't in use as far as I can tell, should this just be removed? (And the only windows change would then be -xl -> -8c)

Reply:
Thanks for catching that. Yeah, this PR doesn't bump up builders but I expect to follow up with an increase in windows core counts presuming this is a win.